ignite/metrics/accuracy.py

```diff
@@ -167,7 +167,7 @@ def update(self, output: Sequence[torch.Tensor]) -> None:
         self._num_examples += correct.shape[0]
 
     @sync_all_reduce("_num_examples", "_num_correct")
-    def compute(self) -> torch.Tensor:
+    def compute(self) -> float:
         if self._num_examples == 0:
             raise NotComputableError("Accuracy must have at least one example before it can be computed.")
-        return self._num_correct.item() / self._num_examples
+        return cast(torch.Tensor, self._num_correct).item() / self._num_examples
```

Comment:
Looks like here it can be done in the same way. We can remove initializations inside the constructor and even remove this cast.

Reply:
Sorry, didn't see your comment, but now I added your suggestion 😄